scripts/pgadmin-link.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+PGADMIN_USERNAME=someone@something.com
+PGADMIN_PASSWORD=QYfzfZTeF8pVMwECvt7jgBdk
```

Comment:
Should we use something like https://unix.stackexchange.com/questions/230673/how-to-generate-a-random-string instead. Otherwise anyone can login to our pgadmins.

Reply:
My point is not to hard-code any credentials in our repos. So I would replace the credential with some bash snippet generating a random pw that is also printed to the console for the user of the script to put in their pw manager.

Reply:
Duly noted! I've updated accordingly :-)